graphix/command.py

```diff
@@ -0,0 +1,113 @@
+"""Data validator command classes."""
+
+from pydantic import BaseModel
+from typing import Union, Literal, List, Tuple
+import enum
+
+Node = int
+Plane = Union[Literal["XY"], Literal["YZ"], Literal["XZ"]]
```

Comment:
Can be written `Literal["XY", "YZ", "XZ"]`. But may we use `pauli.Plane` instead?